static/src/javascripts/projects/common/modules/experiments/tests/remind-me-email.js

```diff
@@ -0,0 +1,194 @@
+                        dataLinkName: 'one-off_series-description_blind-date'
+                    },
+                    experience: {
+                        subtitle: null,
+                        title: 'What is <b>Experience?</b>',
+                        description: 'Every Saturday a Guardian reader shares their unique story',
+                        cta: 'Remind me next Saturday',
+                        dataLinkName: 'one-off_series-description_experience'
+                    }
+                }
+            },
+            weekly: {
+                comingUp: {
+                    blindDate: {
+                        subtitle: 'Coming up in next week&rsquo;s Blind Date',
+                        title: '<b>&lsquo;She didn’t judge me for talking about giraffes&rsquo; necks&rsquo;</b>',
+                        description: 'Did digital communications consultant Lucy, 23, click with waitress Charlie, 20?',
+                        cta: 'Get a weekly reminder',
+                        dataLinkName: 'weekly_coming-up_blind-date'
+                    },
+                    experience: {
+                        subtitle: 'Coming up in next week&rsquo;s Experience',
+                        title: '<b>I was stranded in the wilderness for nine days</b>',
+                        description: '&lsquo;I had lost so much weight that I looked like walking skeleton. I was ready to give up&rsquo;',
+                        cta: 'Get a weekly reminder',
+                        dataLinkName: 'weekly_coming-up_experience'
+                    }
+                },
+                seriesDescription: {
+                    blindDate: {
+                        subtitle: null,
+                        title: 'What is <b>Blind Date?</b>',
+                        description: 'Every Saturday we fix up two Guardian readers',
+                        cta: 'Get a weekly reminder',
+                        dataLinkName: 'weekly_series-description_blind-date'
+                    },
+                    experience: {
+                        subtitle: null,
+                        title: 'What is <b>Experience?</b>',
+                        description: 'Every Saturday a Guardian reader shares their unique story',
+                        cta: 'Get a weekly reminder',
+                        dataLinkName: 'weekly_series-description_experience'
+                    }
+                }
+            }
+        };
+
+        function isBlindDate() {
+            return window.guardian.config.page.series === 'Blind date';
+        }
+
+        function isExperience() {
+            return window.guardian.config.page.series === 'Experience';
+        }
+
+        function insertSignupBox(templateData) {
+            var params = defaults({
+                svg: svg(envelope, ['submit-input__icon'])
+            }, templateData);
+
+            var $signupBox = $.create(template(remindMeEmailSignup, params));
```

Comment:
There's no obvious parent to set `.innerHTML` on, so I'd have to use native `.insertBefore()` which requires a DOM element anyway (rather than markup). It would seem from [this](https://github.com/ded/bonzo#bonzoinsertbeforetarget) that I could pass markup to bonzo and it would automatically turn it into a DOM node for insertion, but having tried it just now, it didn't work (it's still a string by the time it's trying to put it into the DOM). And [looking at the bonzo code](https://github.com/ded/bonzo/blob/master/src/bonzo.js#L311-L322), it makes sense because I can't see any logic to create elements on the fly if you pass markup into bonzo. Guess the documentation is wrong.

So, yet another reason to hate `bonzo`.